ansible/playbooks/tasks/rebuild-stonith.yaml

```diff
@@ -42,4 +42,20 @@
       op monitor interval="600" timeout="15"
   register: create_stonith
   changed_when: create_stonith.rc == 0
+  when: ansible_distribution_major_version != '16'
+  notify: Write stonith status file
+
+- name: Create new SBD stonith device (sle16)
+  ansible.builtin.command:
+    cmd: >-
+      sudo
+      crm configure primitive
+      stonith-sbd
+      stonith:fence_sbd
```

Comment:
seems like only one line is different, think about using facts like https://github.com/SUSE/qe-sap-deployment/pull/414/files#diff-ad47a045cdb46f8c69019b384291bb487a10c8bd55d474524fd85a813875f16dR162-R168